snp_motion_planning/src/planning_server.cpp

```diff
@@ -43,7 +43,7 @@ static const std::string COLLISION_OBJECT_TYPE_PARAM = "collision_object_type";
 static const std::string OCTREE_RESOLUTION_PARAM = "octree_resolution";
 //   Task composer
 static const std::string TASK_COMPOSER_CONFIG_FILE_PARAM = "task_composer_config_file";
-static const std::string TASK_NAME_PARAM = "task_name";
+static const std::string RASTER_TASK_NAME_PARAM = "raster_task_name";
```

Comment:
Let's change this parameter in a future PR where we introduce the similar parameter for the freespace planner